### ISSUE TITLE
Any interested in adding github actions

### ISSUE THREAD

**Issue by fire** [maiself/godot-python-extension#7]
I can work on adding github actions

**Comment by maiself** [comment]
Sure! There's already an action for triggering a build manually, but its pretty basic and I'm no expert in setting up actions. What are you thinking of adding?

**Comment by fire** [comment]
I didn't see there were actions, because they were manual. hmm

**Comment by maiself** [comment]
Should they be made automatic? I'm not all that familiar with the implications, I remember being concerned about usage limits and trying to keep build times down (looks like I got single source builds down to 3-5 min for each platform...)

The actions I did setup were primarily to give me a sane reliable way to build the extension that would run on systems other than my own (rolling release arch linux isn't good for redistributable builds) and to serve as a very basic build guide for others to use until I can get documentation written.

I'm perfectly fine with having things changed and letting you take lead on this if you interested, as well as leaving things as they are if you aren't, no pressure either way. I'm just glad there's interest enough to have discussion.
